lib/modules/manager/sbt/extract.ts

```diff
@@ -357,9 +482,133 @@ export function extractPackageFile(content: string): PackageFile | null {
     isMultiDeps: false,
     scalaVersion: null,
     variables: {},
+    globalVariables: {},
+    localVariables: {},
+    ...defaultAcc,
   };
 
   // TODO: needs major refactoring?
   const res = lines.reduce(parseSbtLine, acc);
   return res.deps.length ? res : null;
 }
+
+function prepareLoadPackageFiles(
+  _config: ExtractConfig,
```

Comment:
This unused parameter isn't necessary here. `extractPackageFile` or `extractAllPackageFiles` must declare it as the first parameter since it's the part of external contract.